@cubejs-client/core import @cubejs-client/react under an Angular project

Hello 

I have installed all Angular packages and imported into my app.module but the compiler return me this error, why? 

```
ERROR in ../../node_modules/@cubejs-client/core/index.d.ts:11:34 - error TS2307: Cannot find module '@cubejs-client/react' or its corresponding type declarations.

11   import type { ChartType } from '@cubejs-client/react';
```

I wouldn't install others packages for another Framework to have my Angular application working correctly. In this case the application needs the type definition because your package needs it

```
declare module '@cubejs-client/core' {
  import type { ChartType } from '@cubejs-client/react';
```

Why this guide didn't tell this? https://statsbot.co/blog/angular-dashboard-tutorial-with-cubejs/

**Version:**
```
"@angular/core": "~10.0.12",
"@cubejs-client/ngx": "^0.26.61",
"@ngrx/effects": "^10.0.0",
```

Thank you

--- a/packages/cubejs-client-core/index.d.ts
+++ b/packages/cubejs-client-core/index.d.ts
@@ -8,8 +8,6 @@
  */
 
 declare module '@cubejs-client/core' {
-  import type { ChartType } from '@cubejs-client/react';
-
   export type TransportOptions = {
     /**
      * [jwt auth token](security)
@@ -962,6 +960,8 @@ declare module '@cubejs-client/core' {
    * @hidden
    */
   export type TSourceAxis = 'x' | 'y';
+
+  export type ChartType = 'line' | 'bar' | 'table' | 'area' | 'number' | 'pie';
 
   export type TDefaultHeuristicsOptions = {
     meta: Meta;

